composer.json

```diff
@@ -47,6 +47,7 @@
         "symfony/finder": "^4.4 || ^5.2",
         "symfony/form": "^4.4",
         "symfony/framework-bundle": "^4.4",
+        "symfony/http-client": "^4.4 || ^5.2",
```

Comment:
this dependency is already on require dev

Reply:
Removed from `require-dev`.